data/paver/skeleton/__init__.py

```diff
@@ -1,4 +1,5 @@
 import os
```

Comment:
Most of the time, when we create a new MAGFest-related Sideboard plugin, we start by just copying over the files from a different plugin.  However, there is a ``paver create_plugin`` task which will generate a skeleton plugin.  We use that a lot at my job, so we've kept it up to date.  The next several changed files are all feature additions to the skeleton plugin.